package.json

```diff
@@ -33,41 +33,41 @@
     ],
     "license": "MIT",
     "dependencies": {
-        "@nestjs/common": "9.4.2",
-        "@nestjs/core": "9.4.3",
+        "@nestjs/common": "^10.0.3",
+        "@nestjs/core": "^10.0.3",
         "class-transformer": "0.5.1",
         "class-validator": "^0.14.0",
-        "dotenv": "16.0.3",
+        "dotenv": "^16.3.1",
         "lodash": "4.17.21",
         "reflect-metadata": "0.1.13",
         "rxjs": "7.8.1",
-        "tslib": "2.6.0",
-        "typescript": "~5.1.3"
```

Comment:
missing typescript in dependencies

Reply:
Is typescript referenced directly?

sorry, moved to devDependencies, so it looks like no issues were found.